_blog.yml

```
@@ -6541,3 +6541,14 @@
     - llm
     - evaluation
     - agents
+
+- local: generate-images-with-claude-and-mcp
+  title: Generate Images with Claude and Hugging Face
+  author: evalstate
+  thumbnail: /blog/assets/generate-images-with-claude-and-mcp/thumbnail.png
+  date: Aug 14, 2025
+  tags:
+    - mcp
+    - llm
+    - qwen
+    - text_to_image
```

```suggestion
    - text-to-image
```

This one is more used.